TradeItIosTicketSDK2/TradeItConnector.swift

```diff
@@ -31,7 +31,11 @@ internal extension TradeItConnector {
         withCompletionBlock completionBlock: @escaping (TradeItResult, String?) -> Void
     ) {
         DispatchQueue.global(qos: .userInitiated).async {
-            let session = URLSession.shared
+            let config = URLSessionConfiguration.default
+            config.requestCachePolicy = .reloadIgnoringLocalCacheData
+            config.urlCache = nil
+            let session = URLSession.init(configuration: config)
+            
```

Comment:
Can you do some research and see if this only needs to be created once? If so, we shouldn't create it every request. Also can you confirm that it won't appear in `Cache.db`